beacon-chain/blockchain/process_block.go

```diff
@@ -413,6 +423,10 @@ func (s *Service) onBlockBatch(ctx context.Context, blks []interfaces.SignedBeac
 			}
 		}
 		s.saveInitSyncBlock(blockRoots[i], b)
+		if err = s.handleBlockAfterBatchVerify(ctx, b, blockRoots[i], fCheckpoints[i], jCheckpoints[i]); err != nil {
```

Comment:
What is this change?

Reply:
To cache justified and finalized block hashes before calling forkchoice updated